### PR TITLE
added example notebooks for pylibCZIrw

### PR DESCRIPTION
**Pull request by CraigZeiss**
added example notebooks for pylibCZIrw

### CODE REVIEW

File: jupyter_notebooks/pylibCZIrw/pylibCZIrw_4_0_0.ipynb [hunk 1]
```diff
@@ -0,0 +1,1276 @@
...
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Install pylibCZIrw and dependencies\n",
+    "! pip install --upgrade pip\n",
+    "! pip install \"pylibCZIrw>=4.0\" \"cztile>=0.0,<1.0\" matplotlib tqdm scikit-image pooch requests"
```
Comment (soyers) [inline jupyter_notebooks/pylibCZIrw/pylibCZIrw_4_0_0.ipynb:16]: Can we restrict the requirements to alsway coose 4 as a major version? E.g. pylibCZIrw~=4.0

**Review by soyers — CHANGES_REQUESTED** [review]

File: jupyter_notebooks/pylibCZIrw/pylibCZIrw_4_0_0.ipynb (hunk 1 above)
Comment (CraigZeiss) [reply jupyter_notebooks/pylibCZIrw/pylibCZIrw_4_0_0.ipynb]: done

**Review by soyers — APPROVED** [review]

**Review by soyers — APPROVED** [review]

**Review by soyers — APPROVED** [review]
LGTM

**Comment by zeissmicroscopy** [comment]
Fine for me AFAIK.

**Review by zeissmicroscopy — APPROVED** [review]
As discussed in teams.

**Pull request merged**